grant/projects/subview.md

```diff
@@ -0,0 +1,99 @@
+# subview.xyz
+
+## **Project Description**
+
+Subview is blockchain explorer for inspecting and analyzing EVM Chains. Subview provides a comprehensive, easy-to-use interface for users to view, confirm, and inspect transactions on EVM (Ethereum Virtual Machine).
+
+### **Why is this project is good for the Crab ecosystem:**
+
+1. Crab DVM currently does not have a specific DVM block explorer. For developers, will encounter obstacles when using the Crab network, especially when developers develop a Dapp. Subview can provide detailed block information and transaction information，developers can read and verify Solidity smart contracts and access pre-existing contracts to fast-track development.
```

Comment:
I see subscan also explore some CSC data, like: https://crab.subscan.io/account/0x0050f880c35c31c13bfd9cbb7d28aafaeca3abd2?tab=contract

So, what's the difference between you two?

Reply:
After reading the comparison, the contract analysis part is similar to the subview. Contract parsing is also the most basic part.

diff:
https://subview.xyz/tokens // Tokens can count the existence of all tokens in the crab smart chain, which is particularly useful.

https://subview.xyz/token/0x8eb2281bbD90ea671C9f6Cb1e51102B7FE3D480e/token-transfers // Comprehensive ERC20 token analysis

https://subview.xyz/token/0x3788df4fdC026f5Ea91a333fCf7CeD7a52c92471/token-transfers // ERC721 token analysis